docs/source/install/env-setup.rst

```diff
@@ -31,6 +31,9 @@ Install Dependencies
 .....................
 Users should install PyTorch v2.1.0 and DGL v1.1.3 that is the core dependency of GraphStorm using the following commands.
```

Comment:
Is this our "default" guideline for new users? Is there a reason to recommend DGL v1 and an older PyTorch version?